konfera/views.py

```diff
@@ -1,6 +1,9 @@
+from django.contrib import messages
+from django.http import HttpResponseRedirect
 from django.shortcuts import render, get_object_or_404
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
```

Comment:
You should put a blank line between each group of imports (between django and konfera).

https://www.python.org/dev/peps/pep-0008/#imports